cocos/asset/asset-manager/downloader.ts

```diff
@@ -25,10 +25,10 @@
 import { BUILD, EDITOR, EDITOR_NOT_IN_PREVIEW } from 'internal:constants';
 import { cclegacy } from '@base/global';
 import { js } from '@base/utils';
+import { callInNextTick } from '@pal/utils';
 import { sys, misc, path } from '../../core';
 import Cache from './cache';
-import downloadFile from './download-file';
-import { FileProgressCallback } from './download-file';
+import downloadFile, { FileProgressCallback } from './download-file';
```

Comment:
I wanna separate these two import statement, but the current eslint `import/no-deplicates` rule has no suitable options

https://github.com/import-js/eslint-plugin-import/blob/v2.27.5/docs/rules/no-duplicates.md